osu.Server.Spectator/Hubs/MultiplayerHub.cs

```diff
@@ -515,6 +516,16 @@ private void ensureValidStateSwitch(MultiplayerRoom room, MultiplayerUserState o
         {
             switch (newState)
             {
+                case MultiplayerUserState.BeatmapNotDownloaded:
```

Comment:
probably just combine with idle below, since they are basically the same thing.

Reply:
done.